### DISCUSSION TITLE
Question: Blue Argon Err. On InboxSDK

### DISCUSSION THREAD

**Discussion by DevEtaiGabbai** [InboxSDK/InboxSDK#1131] [Q&A] [answered]
**Question(s)**
Hey,
I recently received a 'Blue Argon' rejection notice from the CWS. 

Here is the violating code:
const a = { load: (e, n) => Promise.resolve().then((() => { if (!t.__InboxSDKImpLoader) return a._loadScript().then((() => { if (!t.__InboxSDKImpLoader) throw new Error("Implementation file did not load correctly") })) })).then((() => t.__InboxSDKImpLoader.load("0.1", e, n))), _loadScript: (0, i.default)((function() { return (0, o.default)("https://www.inboxsdk.com/build/platform-implementation.js", { nowrap: !0 }) })), preload() { a._loadScript() }

I don't know if this is because I possibly built with webpack dev configs, or if it's something on InboxSDK's end.

Any help is appreciated. 
Thanks,
Etai

**Comment by wegry** [comment, marked as answer]
Hi @DevEtaiGabbai, it looks like you might be using an older version of `@inboxsdk/core`. Using the latest version should not include the offending code. https://github.com/InboxSDK/InboxSDK/pull/970 is related.
